bears/general/AnnotationBear.py

```diff
@@ -0,0 +1,72 @@
+from coalib.parsing.StringProcessing.Core import (search_for,
+                                                  search_in_between,
+                                                  unescaped_search_in_between)
+from coalib.bearlib.languages.LanguageDefinition import LanguageDefinition
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.HiddenResult import HiddenResult
+
+
+class AnnotationBear(LocalBear):
+
+    def run(self, filename, file, language: str, language_family: str):
+        """
+        Finds out all the positions of strings and comments in a file.
+        This does not check however for comments within strings and
+        strings within comments.
+
+        :param language:        Language to be whose annotations are to be
+                                searched.
+        :param language_family: Language family whose annotations are to be
+                                searche.
+
+        :return:                HiddenResult with content conatining
+                                a tuple of tuples of the form
+                                (annotation_start, annotation_end).
+        """
+
+        text = ''.join(file)
+        lang_dict = LanguageDefinition(language, language_family)
+
+        # Strings
+        strings = dict(lang_dict["string_delimiters"])
+        strings.update(lang_dict["multiline_string_delimiters"])
+        matches_found = find_multiline(text, strings, escape=True)
```

Comment:
@sils1297  the thing is, there's no such thing as a single line string right? i mean people can add \ and extend the string onto another line right?